e2e/rbd.go

```diff
@@ -349,6 +350,228 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a PVC and bind it to an app using rbd-nbd mounter", func() {
```

Comment:
can you please enable this E2E also https://github.com/ceph/ceph-csi/blob/devel/e2e/rbd.go#L485-L496?

Reply:
Cool